Makefile

```diff
@@ -70,16 +83,21 @@ vendor:
 	@echo "✅ vendor"
 .PHONY: vendor
 
-run: $(OPERATOR_SDK) gen
-	$(OPERATOR_SDK) run --local --operator-flags "operator run"
-.PHONY: run
+run: gen
+	go build -o $(NOOBAA_OPERATOR_LOCAL) -mod=vendor $(CMD_MANAGER)
+	$(BIN)/noobaa-operator-local operator run
+	.PHONY: run
```

Comment:
```suggestion
.PHONY: run
```

Reply:
```suggestion
.PHONY: gen
```

Reply:
@liranmauda why `.PHONY: gen`?

Reply:
@dannyzaken @v-harihar 
Sorry, you are correct, my mistake 😄 
I thought that the target is gen... and the target is run...